_episodes/21-carpentries.md

```diff
@@ -14,40 +15,53 @@ keypoints:
 
 ---
 
-In becoming a Carpentry instructor,
+In becoming a certified [Carpentry instructor](https://carpentries.org/instructors/),
 you are also becoming part of a community of like-minded volunteers.
-This section provides some background on both projects and information
-about how you can get involved.
-
-## History
-
-[The Carpentries](https://carpentries.org/) is a fiscally sponsored project of [Community Initiatives]()
-that was founded on 1 January, 2018. The Carpentries project comprises [Software Carpentry]({{ site.swc_site }}) and [Data Carpentry]({{ site.dc_site }}) communities of Instructors, Trainers, Maintainers, helpers, and supporters who share a mission to teach foundational computational and data science skills to researchers. 
-
-[Software Carpentry]({{ site.swc_site }}) was founded in 1998 with the mission of teaching lab skills
-for research computing. [Data Carpentry]({{ site.dc_site }}) was founded in 2013 with the mission of building communities teaching universial data literacy. While one focuses on computing and the other on data, both organizations share the same value of promoting efficient, shareable, and reproducible research practices. Their aligned misions are accomplished by running accessible, inclusive training workshops; teaching openly available, high-quality, domain-tailored lessons; and fostering an active, inclusive, diverse instructor community that promotes and models reproducible research as a community norm.
-
-You can learn more about the history
-and goals of each project by reading
-"[Software Carpentry: Lessons Learned](https://f1000research.com/articles/3-62/v2)" and
-"[Data Carpentry: Workshops to Increase Data Literacy for Researchers](http://ijdc.net/index.php/ijdc/article/view/10.1.135)".
-
-Similarities between Software Carpentry and Data Carpentry workshops include:
-*   a focus on computational skills.
+This section provides some background on 
+[The Carpentries]({{ site.carpentries_site }}) projects and information
+about how to get involved.
+
+## A Brief History
+
+[Software Carpentry]({{ site.swc_site }}) was founded in 1998 
+with the mission of teaching lab skills for research computing. 
+[Data Carpentry]({{ site.dc_site }}) was founded in 2014 with the mission of building communities teaching universial data literacy. 
+Also in 2014, [Library Carpentry](({{ site.lc_site }})) was founded with the mission of teaching data skills to people working in library- and information-related roles.
+
+One January 1, 2018, Software Carpentry and Data Carpentry merged 
+their projects to form a new project called
+[The Carpentries]({{ site.carpentries_site }})
+under the fiscal sponsorship of 
+[Community Initiatives](https://communityin.org/)
+On November 1, 2018, The Carpentries Executive Council 
+approved [Library Carpentry](({{ site.lc_site }})) 
+as an official Lesson Program.
```

Comment:
I would modify this to "... approved Library Carpentry as the third official Lesson Program of the Carpentries."

Reply:
👍